scripts/tests.bat

```diff
@@ -32,9 +32,8 @@ if "%TESTS%"=="On" (
 
     REM Copy the DLLs into the test directory which need to be able to run.
     cd %APPVEYOR_BUILD_FOLDER%\build\test\%CONFIGURATION%
-    copy %APPVEYOR_BUILD_FOLDER%\build\evmjit\libevmjit\%CONFIGURATION%\evmjit.dll .
     copy "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\redist\x86\Microsoft.VC140.CRT\msvc*.dll" .
-    copy %ETHEREUM_DEPS_PATH%\win64\bin\OpenCl.dll .
+    copy %APPVEYOR_BUILD_FOLDER%\build\libaleth-interpreter\%CONFIGURATION%\aleth-interpreter.dll .
```

Comment:
I don't think we run any tests for this DLL. I'm surprised this is even built.

Reply:
For some reason we build with `-DALETH_INTERPRETER_SHARED=ON`

Should we better statically link instead?

Reply:
The name is confusing, but `ALETH_INTERPRETER_SHARED=ON` build DLL additionally. Still aleth-interpreter is statically linked to aleth / testeth. This build tests if the DLL can be linked but no test are executed.

To test statically linked interpreter you need `testeth -- --vm interpreter`.
To test DLL you need `testeth -- --vm path/aleth-interpreter.dll`.

Reply:
We don't run tests to safe time.

Reply:
I'm suggesting to remove this line, because in case we stop building interpreter.dll the script will stop working again.

Reply:
I'm suggesting to remove this line, because in case we stop building interpreter.dll the script will stop working again.

Reply:
It won't stop because of the failed copy, it doesn't stop now when it can't copy `OpenCl.dll`